wiki/Dark-Mode.md

```diff
@@ -81,26 +78,39 @@ _Note:_
+- Try to introduce similar colors for the same UI. For example: if you are using `Topic Name` somewhere then if possible try to use similar colors which have already been used for topic names in the android app.
+- If you are looking for an already existing view and trying to find its light/dark color code, you will need to navigate to [component_colors](https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/component_colors.xml) first and find the name for the view in component_colors.xml file. For example you can look at:
+- - [<color name="component_color_add_profile_activity_info_icon_color">@color/color_palette_info_icon_color</color>](https://github.com/oppia/oppia-android/blob/12ded96886d29d46682e837c3232c4b6d3377800/app/src/main/res/values/component_colors.xml#L124C3-L124C114)
+- - In this, we now we can find the related color_palette name, i.e., color_palette_info_icon_color. Now we can visit the color_palette file for light and dark mode, i.e.,for light mode [<color name="color_palette_info_icon_color">@color/color_def_chooser_grey</color>](https://github.com/oppia/oppia-android/blob/12ded96886d29d46682e837c3232c4b6d3377800/app/src/main/res/values/color_palette.xml#L65) and for dark-mode [<color name="color_palette_info_icon_color">@color/color_def_oppia_silver</color>](https://github.com/oppia/oppia-android/blob/12ded96886d29d46682e837c3232c4b6d3377800/app/src/main/res/values-night/color_palette.xml#L60)
+- - In the end, we can see that the light color is color_def_chooser_grey and dark color is color_def_oppia_silver and we can find the exact color code for both in the [color_def](https://github.com/oppia/oppia-android/blob/develop/app/src/main/res/values/color_defs.xml) file.
+
+### Notes for Reviewers
+- The naming of new colors across all files should follow the rules mentioned [above](https://github.com/oppia/oppia-android/wiki/Dark-Mode#knowing-the-convention).
+- If any new color is getting introduced in the `color_defs.xml` file, then treat it as a red flag and keep a close eye on why it is getting introduced? Why can't any of the existing colors be used? Has this been approved by the design team?
+- If any new color palette is getting introduced in `color_defs.xml` keep a close eye on that to make sure that there is no similar color palette already defined in `color_palette.xml` light/dark files. If there is a similar color palette, can you rename the existing color_palette such that it can be used on existing places as well as new places?
+- Other than thumbnails and icons in `mipmap` and `drawable` folder, all other places should reference color code from `component_colors.xml` file only. There should not be any direct usage of color from `color_palette.xml` or `color_defs.xml` or any other file.(Note: there is an issue filed at #5114 to do this automatically via a CI check, but until that issue is resolved, this needs to be checked manually.)
```

Comment:
Add space before '('.

Can you link the issue properly (using markdown)? Should be 
```
[#5114](https://github.com/oppia/oppia-android/issues/5114)
```

Reply:
Applied both changes.